src/rt/cover.c

```diff
@@ -1664,6 +1715,13 @@ static int cover_append_to_chain(cover_chain_t *chain, cover_tag_t *tag,
    return rv;
 }
 
+static bool cover_tag_unreachable(cover_report_ctx_t *ctx, cover_tag_t *tag)
+{
+   if ((ctx->tagging->mask & COVER_MASK_EXCLUDE_UNREACHABLE) == 0)
+      return false;
+   return (tag->data & COV_FLAG_UNREACHABLE);
```

Comment:
```suggestion
   return !!(tag->data & COV_FLAG_UNREACHABLE);
```

Never store or return a `bool` value that's not zero or one. Usually `bool` is typedef-ed to `char` so it can't store `COV_FLAG_UNREACHABLE` (1 << 25). It works here because it happens to always be in a register but if the result were spilled to the stack or stored in a struct or global variable, a true value would mysteriously become false which is very hard to debug.